server/middlewares/csp.js

```diff
@@ -5,7 +5,12 @@ let trusted = [
 ];
 
 if (process.env.NODE_ENV !== 'production') {
+  let host = 'http://'
+             + (process.env.HOST || 'localhost')
+             + ':'
+             + (process.env.SYNC_PORT || process.env.PORT || '3000');
```

Comment:
indenting should be two space.

It may be cleaner to put the save the individual peaces into separate const's and than use a templete string to put it together:

```
trusted = trusted.concat([
     `http://${host}:{port}`,
      ... 
    ]);
```

You would also need to update the websocket url with this